Apps/W1/AMCBanking365Fundamentals/app/Codeunits/AMCBankingMgt.Codeunit.al

```diff
@@ -623,5 +660,12 @@ codeunit 20105 "AMC Banking Mgt."
         else
             exit(BankAccount."AMC Bank Name" + FileExtTxt);
     end;
+
+    [NonDebuggable]
```

Comment:
The code is open sourced, not sure if `NonDebuggable`  would be very helpful

Reply:
Just remove the NonDebuggable